tests/test_Config.py

```diff
@@ -115,6 +115,27 @@ def tearDown(self):
         del self.outer
         del self.comp
 
+    def testFieldTypeAnnotationRuntime(self):
+        # test parsing type annotation for runtime dtype
+        testField = pexConfig.Field[str](doc="")
+        self.assertEqual(testField.dtype, str)
```

Comment:
Have you tested with `from __future__ import annotations`, too?  Or is that now the default in 3.10, and the question is really whether his works back in 3.7 or 3.8 (and presumably the test matrix will tell us if it doesn't)?

Reply:
differed type evaluation is not baseline in python 3.10. All of our files still need `from __future__ import annotations`. The thing python 3.10 brings us is generics on built in types. Differed annotations will not happen until 3.11 at the earliest, but even that I think is slipping because people cant come to a consensus on things.